util/src/length_prefix_encoding/mod.rs

```diff
@@ -1,4 +1,2 @@
-/// Module that handles decoding functionality
 pub mod decoder;
-/// Module that handles encoding functionality
 pub mod encoder;
```

Comment:
Why are those doc strings removed?

Reply:
@aparcar I'll consider this resolved for now :)

It would be possible to move the first line of the new summary into the `mod.rs` file, but it consists of two lines that you wouldn't look at when working on the docs, so that doesn't seem particularly useful here.